online-shop/src/app/components/product-details/product-details.component.html

```diff
@@ -0,0 +1,26 @@
+<div *ngIf="product" class="product-details">
+  <div class="details-container">
+    <h1>Product: {{product?.name}}</h1>
+    <button (click)="addToCart(product)" *ngIf="authService.isCustomer()" class="default-border">Add to cart</button>
+    <button (click)="edit()" *ngIf="authService.isAdmin()" class="default-border" id="edit-button">Edit</button>
+    <button (click)="delete()" *ngIf="authService.isAdmin()" class="default-border" id="delete-button">Delete</button>
+    <div class="default-border product-information">
+      <div class="divTable properties">
```

Comment:
Please use \<table\> and the associated table tags (tr, td , th) instead of \<div\>